glean-core/rlb/tests/schema.rs

```diff
@@ -88,8 +89,62 @@ fn validate_against_schema() {
     };
     let _ = new_glean(Some(cfg));
 
-    // Define a new ping and submit it.
     const PING_NAME: &str = "test-ping";
+
+    // Test each of the metric types, just for basic smoke testing against the
+    // schema
+
+    // TODO: 1695762 Test all of the metric types against the schema from Rust
+
+    let rate_metric: RateMetric = RateMetric::new(CommonMetricData {
+        name: "rate".into(),
+        category: "test".into(),
```

Comment:
Can we put the `category` first so I can read it in the same order ( `test.rate` ) that it appears in the payload?

Reply:
Sure thing.